test/http-server-v1.js

```diff
@@ -4,13 +4,27 @@ require('./init');
 require('should');
 const HttpTestUtils = require('./utils/http');
 const STATUS_CODES = require('../lib/control/util/status-codes');
+const STATUS = require('../lib/utils/status');
 const testServerPort = 3000;
 
+const goodDefaultToken = () => Promise.resolve({
+  status: STATUS.READY,
+  data: {
+    token: 'sometoken'
+  }
+});
+
 class MockProvider {
   constructor() {}
 }
 
 class StorageServiceMock {
+  constructor() {
+    this.defaultToken = {
+      initialize: goodDefaultToken
```

Comment:
Do we want to include a test for the negative case too e.g. have initialize throw and ensure we get a "service unavailable" response from the health end point?